lib/interface/cli/helpers/logs.js

```diff
@@ -53,6 +56,14 @@ async function _fallbackLogs(workflowId, interval, retriesLeft) {
 }
 
 const followLogs = async (workflowId) => {
+    try {
+        checkVersion(nodeVersionToValidate);
+    } catch (error) {
+        // eslint-disable-next-line max-len
+        console.log(`As your node version (${process.version}) is not compatible with supported node version (${nodeVersionToValidate}) for this command, you won't be able to see logs`);
```

Comment:
I say, remove the `As`, `Node version x.x.x is not compatible with build logs command, click here to view the logs in the Codefresh UI:` and then build a url using the `context.url` and the `workflowId`

Reply:
and also maybe add a * or two and the beginning, not sure about that, maybe we should ask idan 🤔

Reply:
done